lighthouse-core/gather/gatherers/link-elements.js

```diff
@@ -147,6 +154,29 @@ class LinkElements extends Gatherer {
 
     return linkElements;
   }
+
+  /**
+   * @param {LH.Gatherer.PassContext} passContext
+   * @param {LH.Gatherer.LoadData} loadData
+   * @return {Promise<LH.Artifacts['LinkElements']>}
+   */
+  async afterPass(passContext, loadData) {
+    return await this._getArtifact(passContext, loadData.networkRecords);
+  }
+
+  /**
+   * @param {LH.Gatherer.FRTransitionalContext<'DevtoolsLog'>} passContext
+   * @return {Promise<LH.Artifacts['LinkElements']>}
+   */
+  async getArtifact(passContext) {
```

Comment:
FWIW in FR world these aren't really passes anymore. When renaming the methods and copy/pasting implementations I've been skipping this rename, but if you write new methods we can just say `context` (or come up with a more appropriate name, `phaseContext` maybe?)